### PR TITLE
Add Ember v2.7 and v2.8 support, tests with multiple versions of Ember, fix #91

### LINKED ISSUE #91: Doesn't work with ember-templates-compiler 2.7.2, but works with 2.3.0
Got an error `TypeError: (module.exports || exports).precompile is not a function`

### PR DESCRIPTION
**Pull request by zoltan-nz**
* support Ember v2.7 and v2.8
* extend tests
* bump package version to v1.1.0

Fixing: https://github.com/dgeb/grunt-ember-templates/issues/91

### CODE REVIEW

**Comment by zoltan-nz** [comment]
Hi @dgeb, could you please review if you will have a sec? Cheers.  :)

**Comment by dgeb** [comment]
@zoltan-nz quite the delta, but looks good as far as I can tell. :+1:

File: tasks/ember_templates.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function manualCompile(templateCompilerPath, template, grunt){
   vm.runInContext(templateCompilerJs, context, 'ember-template-compiler.js');
 
   // Compile the template
-  vm.runInContext('compiledJS = (module.exports || exports).precompile(template);', context);
+  vm.runInContext('compiledJS = ((typeof Ember !== "undefined" && Ember.HTMLBars) || module.exports || exports).precompile(template);', context);
```
Comment (zoltan-nz) [inline tasks/ember_templates.js]: @dgeb Actually, the real change is only this few characters, however testing with different versions with Ember is brute force, the expected result files are massive, but at least the test emulate a realistic environment. 

I guess, there will be a big change to keep the compatibility with Ember v2.9. :)

Comment (dgeb) [inline tasks/ember_templates.js]: Ah, thanks for highlighting that change @zoltan-nz.

Yes, there will be a much more significant delta for Ember 2.9 :)

**Pull request merged**